CHANGES.txt

```diff
@@ -6,14 +6,27 @@
   * Add C++20 keywords guarded by PROTOBUF_FUTURE_CPP20_KEYWORDS
   * Fixed crash in ThreadLocalStorage for pre-C++17 compilers on 32-bit ARM.
   * Clarified that JSON API non-OK statuses are not a stable API.
+  * Added a default implementation of MessageDifferencer::Reporter methods.
+  * proto2::MapPair is now an alias to std::pair.
+  * Hide C++ RepeatedField::UnsafeArenaSwap
+  * Fix an unchecked error return from PushLimit()
```

Comment:
This CL associated with this change was rolled back.

Reply:
Removed, thanks!